sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/src/Generated/Models/UeInfo5G.cs

```diff
@@ -0,0 +1,196 @@
+        /// <param name="pointer"> AMF pointer. </param>
+        /// <param name="mccInfoFivegGutiPlmnMcc"> Mobile country code (MCC). </param>
+        /// <param name="mncInfoFivegGutiPlmnMnc"> Mobile network code (MNC). </param>
+        internal UeInfo5G(RatType ratType, DateTimeOffset? lastReadOn, IDictionary<string, BinaryData> serializedAdditionalRawData, string supi, string pei, IList<UeSessionInfo5G> sessionInfo, string perUeTnla, long? amfUeNgapId, int? ranUeNgapId, string lastVisitedTai, IList<Snssai> allowedNssai, UeState? ueState, RrcEstablishmentCause? rrcEstablishmentCause, UeUsageSetting? ueUsageSetting, DateTimeOffset? lastActivityOn, string ngeNbId, string eNbId, string n3IwfId, string wagfId, string tngfId, string nid, int? bitLength, string gnbValue, string mccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc, string mncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc, string locationType, string tac, string mccInfoConnectionInfoLocationInfoPlmnMcc, string mncInfoConnectionInfoLocationInfoPlmnMnc, int fivegTmsi, int regionId, int setId, int pointer, string mccInfoFivegGutiPlmnMcc, string mncInfoFivegGutiPlmnMnc) : base(ratType, lastReadOn, serializedAdditionalRawData)
+        {
+            Supi = supi;
+            Pei = pei;
+            SessionInfo = sessionInfo;
+            PerUeTnla = perUeTnla;
+            AmfUeNgapId = amfUeNgapId;
+            RanUeNgapId = ranUeNgapId;
+            LastVisitedTai = lastVisitedTai;
+            AllowedNssai = allowedNssai;
+            UeState = ueState;
+            RrcEstablishmentCause = rrcEstablishmentCause;
+            UeUsageSetting = ueUsageSetting;
+            LastActivityOn = lastActivityOn;
+            NgeNbId = ngeNbId;
+            ENbId = eNbId;
+            N3IwfId = n3IwfId;
+            WagfId = wagfId;
+            TngfId = tngfId;
+            Nid = nid;
+            BitLength = bitLength;
+            GNBValue = gnbValue;
+            MccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc = mccInfoConnectionInfoGlobalRanNodeIdPlmnIdMcc;
+            MncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc = mncInfoConnectionInfoGlobalRanNodeIdPlmnIdMnc;
+            LocationType = locationType;
+            Tac = tac;
+            MccInfoConnectionInfoLocationInfoPlmnMcc = mccInfoConnectionInfoLocationInfoPlmnMcc;
+            MncInfoConnectionInfoLocationInfoPlmnMnc = mncInfoConnectionInfoLocationInfoPlmnMnc;
+            FivegTmsi = fivegTmsi;
+            RegionId = regionId;
+            SetId = setId;
+            Pointer = pointer;
+            MccInfoFivegGutiPlmnMcc = mccInfoFivegGutiPlmnMcc;
+            MncInfoFivegGutiPlmnMnc = mncInfoFivegGutiPlmnMnc;
+            RatType = ratType;
+        }
+
+        /// <summary> Initializes a new instance of <see cref="UeInfo5G"/> for deserialization. </summary>
+        internal UeInfo5G()
+        {
+        }
+
+        /// <summary> Subscription Permanent Identifier. </summary>
+        public string Supi { get; set; }
```

Comment:
same question for the naming

Reply:
Name is correct. Its as per telecom specification Sect 2.2A https://www.etsi.org/deliver/etsi_TS/123000_123099/123003/16.06.00_60/ts_123003v160600p.pdf